site/content/en/docs/tasks/run_rayclusters.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Run A RayCluster"
+date: 2024-01-17
+weight: 6
+description: >
+  Run a RayCluster on Kueue.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [RayCluster](https://docs.ray.io/en/latest/cluster/getting-started.html).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
```

Comment:
We should probably document the Kueue version required for this as well as the Kuberay version required (v1.1 which introduced the Suspend field in RayCluster)